tests/roxctl/bats-tests/cluster/central-init-bundles.bats

```diff
@@ -13,15 +13,15 @@ setup_file() {
 }
 
 setup() {
-  output_dir="$(mktemp -d -u)"
+  output="$(mktemp -d -u)"
```

Comment:
How that fixes the issue? I see that `-u` means dry-run, so the proper fix would be:
```suggestion
  output="$(mktemp -d)"
```
Actually all the rest of the old code would be fine. (But maybe I am not seeing something)

Reply:
>      -u, --dry-run
>             Operate in “unsafe” mode.  The temp file will be unlinked before mktemp exits.  This is slightly better >than mktemp(3) but still introduces a race condition.  Use of this option is not
>             encouraged.

Reply:
I think, I am not following it. Keeping the command like this will not create the directory. Do we have other code that would create it for us?